src/main/java/com/derongan/minecraft/mineinabyss/configuration/MIAConfig.kt

```diff
@@ -33,6 +34,10 @@ internal object MIAConfig : IdofrontConfig<MIAConfig.Data>(
         val layers: List<LayerImpl>, //TODO way of changing the serializer from service
         private val hubSectionName: String = "orth",
     ) {
-        val hubSection by lazy { WorldManager.getSectionFor(hubSectionName) }
+        val hubSection by lazy {
+            WorldManager.getSectionFor(hubSectionName).also {
```

Comment:
Elvis operator is useful for this in particular. You can throw an actual error since we really wanna be sure the section is set properly I think. This ensures the type isn't nullable.
```kotlin
val hubSection by lazy { WorldManager.getSectionFor(hubSectionName) ?: error("Section $hubSectionName was not found for the hub.") }
```